setup.py

```diff
@@ -0,0 +1,56 @@
+import ast
```

Comment:
Why are you importing this?

Reply:
That's from a previous version, will remove.